lib/rspecq/queue.rb

```diff
@@ -241,11 +244,37 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    # Returns the threshold of failures that need to take place before the
+    # build is terminated. This configuration is persisted on a per-build basis
+    # on Redis
+    def fail_fast
+      return nil unless published?
+
+      @fail_fast ||= Integer(@redis.hget(key_queue_config, 'fail_fast'))
+    end
+
+    # Returns true if the number of failed tests, has surpassed the threshold
+    # to render the run unsuccessful. This is also used internally to block any
+    # new jobs being enqueued
```

Comment:
The "This is also used" is an implementation detail and a side-effect of this feature, not the primary aim of it. So I believe we should drop it because it's actually a bit confusing.

Instead, I would add:

> [...] the run unsuccessful **and the build should be aborted**.

...to clarify that this method should be used as a signal to abort the build.